### PR TITLE
Create tandoor_recipes.md

### PR DESCRIPTION
**Pull request by Eschguy**
Create example for Tandoor Recipes.

### CODE REVIEW

File: example_configs/tandoor_recipes.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+LDAP_AUTH=1
+AUTH_LDAP_SERVER_URI=ldap://lldap:3890/
+AUTH_LDAP_BIND_DN=uid=ro_admin,ou=people,DC=example,DC=com
+AUTH_LDAP_BIND_PASSWORD=CHANGEME
+AUTH_LDAP_USER_SEARCH_BASE_DN=ou=people,DC=example,DC=com
+AUTH_LDAP_USER_SEARCH_FILTER_STR=(&(&(objectclass=person)(memberOf=cn=tandoor_users,ou=groups,dc=example,dc=com))(uid=%(user)s))
```
Comment (nitnelave) [inline example_configs/tandoor_recipes.md:6]: Maybe add a comment to say that it only whitelists users from the tandoor_users group?

File: example_configs/tandoor_recipes.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+AUTH_LDAP_USER_ATTR_MAP={'first_name': 'givenName', 'last_name': 'sn', 'email': 'mail'}
+AUTH_LDAP_ALWAYS_UPDATE_USER=1
```
Comment (nitnelave) [inline example_configs/tandoor_recipes.md:8]: What does that mean?

File: example_configs/tandoor_recipes.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+LDAP_AUTH=1
+AUTH_LDAP_SERVER_URI=ldap://lldap:3890/
+AUTH_LDAP_BIND_DN=uid=ro_admin,ou=people,DC=example,DC=com
```
Comment (nitnelave) [inline example_configs/tandoor_recipes.md:3]: Small nit: can you use consistent casing for the DC and ou and so on? Maybe everything in lowercase

**Review by nitnelave — APPROVED** [review]

File: example_configs/tandoor_recipes.md (hunk 3 above)
Comment (Eschguy) [reply example_configs/tandoor_recipes.md:10]: Done!

File: example_configs/tandoor_recipes.md (hunk 1 above)
Comment (Eschguy) [reply example_configs/tandoor_recipes.md:19]: Good idea, added better explanations/formatting.

File: example_configs/tandoor_recipes.md (hunk 2 above)
Comment (Eschguy) [reply example_configs/tandoor_recipes.md:29]: Added better explanations document -wide

**Review by Eschguy — COMMENTED** [review]
Made everything more legible and added explanations

**Comment by nitnelave** [comment]
Thank you everyone for the contribution!
This feature is now available in the latest release, [v0.5.0](https://github.com/lldap/lldap/releases/tag/v0.5.0).
You can support LLDAP by starring our repo, contributing some configuration examples and becoming a sponsor.

**Pull request merged**